src/ctia/entity/incident.clj

```diff
@@ -178,7 +183,13 @@
                        ;; we assume this was updated by the status route on Open. will be garbage if status was updated
                        ;; in any other way.
                        (get-in prev [:incident_time :opened])
-                       (get-in incident [:incident_time :closed])))))
+                       (get-in incident [:incident_time :closed]))
+
+      (and (new-status? old-status)
```

Comment:
We need to ask for confirmation of the expected status change scenario.
That's very likely that users will move to "Open: Contained" from another Open status
```suggestion
      (and (or (open-status? old-status) (new-status? old-status))
```